poetry/semver/__init__.py

```diff
@@ -1,6 +1,7 @@
 import re
 
 from .empty_constraint import EmptyConstraint
+from .patterns import COMPLETE_VERSION
```

Comment:
the imports are sorted in this module. though this doesn't seem to be the case in all of them, ordered imports are superior, imo.

Reply:
OK, moving it
```python
from .patterns import CARET_CONSTRAINT
from .patterns import COMPLETE_VERSION
from .patterns import TILDE_CONSTRAINT
```